applications/main/bad_usb/helpers/ducky_script_keycodes.c

```diff
@@ -6,21 +6,16 @@ typedef struct {
     uint16_t keycode;
 } DuckyKey;
 
-static const DuckyKey ducky_keys[] = {
-    {"CTRL-ALT", KEY_MOD_LEFT_CTRL | KEY_MOD_LEFT_ALT},
-    {"CTRL-SHIFT", KEY_MOD_LEFT_CTRL | KEY_MOD_LEFT_SHIFT},
-    {"ALT-SHIFT", KEY_MOD_LEFT_ALT | KEY_MOD_LEFT_SHIFT},
-    {"ALT-GUI", KEY_MOD_LEFT_ALT | KEY_MOD_LEFT_GUI},
-    {"GUI-SHIFT", KEY_MOD_LEFT_GUI | KEY_MOD_LEFT_SHIFT},
-    {"GUI-CTRL", KEY_MOD_LEFT_GUI | KEY_MOD_LEFT_CTRL},
-
+static const DuckyKey ducky_modifier_keys[] = {
```

Comment:
note that this does not enforce constness on `ducky_modifier_keys[]` itself and it will go to .data section. Not quite relevant for FAPs since they are allocated in RAM anyway, but better way would be `static const DuckyKey const ducky_modifier_keys[]`